cli/module_reload_test.go

```diff
@@ -166,11 +180,13 @@ func TestFullReloadFlow(t *testing.T) {
 				map[string]any{
 					moduleFlagPath: manifestPath, generalFlagPartID: "part-123",
 					moduleBuildFlagNoBuild: true, moduleFlagLocal: true,
+					generalFlagNoProgress: true, // Disable progress spinner to avoid race conditions in tests
 				},
 				"token",
 			)
 
-			err = reloadModuleActionInner(cCtx, vc, parseStructFromCtx[reloadModuleArgs](cCtx), logger, false)
+			// Create isolated logger for this subtest
```

Comment:
I see this is done elsewhere -- but do we know what the consequence is of not creating a separate logger? And just using the top-level test logger?